packages/commercetools/theme/pages/Product.vue

```diff
@@ -250,8 +266,11 @@ export default {
       const productChannels = product.value?.availability?.channels?.results ?? [];
       return productChannels;
     });
+    const selectedDelivery = ref(null);
+    const selectDelivery = option => selectedDelivery.value = option;
```

Comment:
To increase readability maybe change this function name to `setSelectedDelivery`, what do you think?

Reply:
done!